travis_after_script.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+if [ "$TRAVIS_PULL_REQUEST" = "false" ]
+then
+docker exec openmrs-referenceapplication cat .OpenMRS/openmrs.log
```

Comment:
Okay. We could fail a test when the server responds with error HTTP status code when accessing the login page. The change would have to be implemented in uitestframework.